app/models/ability/administrator.rb

```diff
@@ -85,6 +85,7 @@ def grant_privileges # rubocop:todo Metrics/AbcSize
 
     # Edit and delete submissions
     can :manage, Submission
+    can :cancel, Submission
   end
```

Comment:
From reading the ticket content:
```
We should probably add can :cancel, Submission to both the lab_manager and sample_management_manager files...
```
I think to add this we would need to add the same change to app/models/ability/lab_manager and sample_management_manager